test/unit/BondingManager.js

```diff
@@ -1,3807 +1,3868 @@
-import Fixture from "./helpers/Fixture"
-import expectRevertWithReason from "../helpers/expectFail"
-import {contractId, functionSig, functionEncodedABI} from "../../utils/helpers"
-import {constants} from "../../utils/constants"
-import math from "../helpers/math"
-import BN from "bn.js"
-import truffleAssert from "truffle-assertions"
-import {assert} from "chai"
-import {ethers, web3} from "hardhat"
-
-import chai from 'chai'
-import {
-  solidity
-} from 'ethereum-waffle'
-
-chai.use(solidity)
-const {
-  expect
-} = chai
-const {DelegatorStatus, TranscoderStatus} = constants
+import Fixture from "./helpers/Fixture";
+// import expectRevertWithReason from "../helpers/expectFail";
+import { contractId, functionSig, functionEncodedABI } from "../../utils/helpers";
+import { constants } from "../../utils/constants";
+import math from "../helpers/math";
+import BN from "bn.js";
+// import truffleAssert from "truffle-assertions";
+import { assert } from "chai";
+import { ethers, web3 } from "hardhat";
+
+import chai from "chai";
+import { solidity } from "ethereum-waffle";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signers";
+
+chai.use(solidity);
+const { expect } = chai;
+const { DelegatorStatus, TranscoderStatus } = constants;
 
 describe("BondingManager", () => {
-    let fixture
-    let bondingManager
+    let fixture;
+    let bondingManager;
 
-    const NUM_ACTIVE_TRANSCODERS = 2
-    const UNBONDING_PERIOD = 2
-    const MAX_EARNINGS_CLAIMS_ROUNDS = 20
+    const NUM_ACTIVE_TRANSCODERS = 2;
+    const UNBONDING_PERIOD = 2;
+    const MAX_EARNINGS_CLAIMS_ROUNDS = 20;
 
-    const PERC_DIVISOR = 1000000
-    const PERC_MULTIPLIER = PERC_DIVISOR / 100
+    const PERC_DIVISOR = 1000000;
+    const PERC_MULTIPLIER = PERC_DIVISOR / 100;
 
-    let accounts
+    let accounts;
```

Comment:
In the other files we call this signers, I think it would be nice if we could stick to that convention.

Reply:
done 👍🏼